plugins/output/adapt/importsource.js

```diff
@@ -484,8 +484,12 @@ function ImportSource(req, done) {
       */
       if(type === 'course') {
         try {
-          var contents = await fs.readFile(path.join(COURSE_ROOT_FOLDER, Constants.Folders.Source, Constants.Folders.Theme, plugindata.theme[0]._theme, 'less', Constants.Filenames.CustomStyle), 'utf8');
-          data = _.extend(data, { tags: formTags, customStyle: contents.toString() });
+          var customStylePath = path.join(COURSE_ROOT_FOLDER, Constants.Folders.Source, Constants.Folders.Theme, plugindata.theme[0]._theme, 'less', Constants.Filenames.CustomStyle);
+          var customStyleContents = '';
+          if (fs.existsSync(customStylePath)) {
```

Comment:
fs.exists is deprecated, fs.existsSync is not (https://nodejs.org/api/fs.html#fs_fs_existssync_path).

I think either it should stay like this or I could put it back to how it was but catch the specific error case of the file not existing rather than rejecting at that point. What do you think?

Reply:
Recommend catching the error.

Reply:
Ok willdo